src/utils.jl

```diff
@@ -42,6 +42,24 @@ end
 
 _debug_mode() = true
 
+
+"""
+    enable_graphics([backend])
+
+Load scripts required to visualize the results of the [`Logger`](@ref).
+
+Calling this function will possibly download and pre-compile a `Makie`
+backend and/or `GraphViz`, so if these are not yet available on your system calling
+`enable_graphics` for the first time may take a while.
+
+Available backends are: `CairoMakie` and `GLMakie`.
+"""
+function enable_graphics(;backend=:GLMakie)
+    supported = (:GLMakie, :CairoMakie)
+    backend in supported || error("supported backends: $supported")
+    @eval Main (DataFlowTasks.@using_opt GraphViz, $backend)
+end
+
```

Comment:
@ffevotte What do you think about a function to simplify the loading of the optional dependencies? This is in line with the other "enablers" we have (e.g. `enable_log` and `enable_debug`).

Reply:
This was a bad idea, so I went back to the original way you were doing things.